Make.inc

```diff
@@ -239,6 +239,8 @@ BUILD_LLVM_CLANG := 0
 BUILD_LLDB := 0
 USE_POLLY := 0
 USE_POLLY_OPENMP := 0
+USE_POLLY_ACC := 0
+CUDALIB_INCLUDE_DIR := /usr/local/cuda/include
```

Comment:
@ararslan NP. I see that Julia uses cygwin to build in Windows. Should paths follow the Unix notation ? with forward slashes between parent and children directories and "\" preceeding spaces ? I'm not familiar with the Cygwin environment.